system/lib/pthread/library_pthread.c

```diff
@@ -31,6 +31,8 @@ char *gets(char *);
 // Extra pthread_attr_t field:
 #define _a_transferredcanvases __u.__s[9]
 
+void __pthread_testcancel();
```

Comment:
Why did `pthread_testcancel` get renamed to `__pthread_testcancel`? Reading here, http://man7.org/linux/man-pages/man3/pthread_testcancel.3.html, it seems that the function should be conformant to POSIX.1-2001, POSIX.1-2008, so it should not need to be prefixed? (Not superfamiliar with the year numbers on POSIX variants, but cross-referencing to pthread_create at http://man7.org/linux/man-pages/man3/pthread_create.3.html, it suggest pthread_testcancel and pthread_create are present in the same versions of the standard)